test/download-data.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This is a simple script that downloads data using s3, this can be extended to simply download data from an HTTP/FTP server.